providers/os/resources/os.lr.manifest.yaml

```diff
@@ -200,15 +200,20 @@ resources:
       expose: {}
       file: {}
       from: {}
-      labels:
-        min_mondoo_version: 9.0.0
+      labels: {}
       run: {}
+      user: {}
     min_mondoo_version: 9.0.0
     refs:
     - title: Dockerfile Reference
       url: https://docs.docker.com/reference/dockerfile/
     - title: Multi-stage builds
       url: https://docs.docker.com/build/building/multi-stage/
+  docker.file.user:
+    fields:
+      group: {}
+      user: {}
+    min_mondoo_version: latest
```

Comment:
```suggestion
    min_mondoo_version: 9.0.0
```

Reply:
Shouldn't this be `11+`? I really don't know how this is set so, I will trust your knowledge on this one. 👍🏽

Reply:
The plugin mechanism has not changed, so older cnspec version can also use the latest plugin

Reply:
Thanks @chris-rock - Updated ✅

Reply:
To expand on what Chris said. In < 9.0 clients you were locked to whatever we shipped in the client binary itself. In 9.x and above we always download the latest provider. So when we ship this it will be downloaded by 9.0+ clients giving us full backwards compat from 9.0 onwards.